src/docs/product/performance/web-vitals.mdx

```diff
@@ -40,23 +38,23 @@ Let’s take a look at the example above which has one unstable element - the bo
 
 ## Other Web Vitals
 
-These Web Vitals are generally less user-visible, but are useful for troubleshooting issues with the Core Web Vitals.
+The below Web Vitals are generally less visible to users, but are useful for troubleshooting issues with the Core Web Vitals.
 
 ### First Paint (FP)
 
-First Paint (FP) measures the amount of time the first pixel takes to appear in the viewport, rendering any visual change from what was previously displayed. This may be in any form from the document object model (DOM), such as background color, canvas, or image. FP helps developers understand if anything unexpected is happening to render the page.
+First Paint, (FP) measures the amount of time it takes for the first pixel to appear in the viewport. This may be in any form from the document object model (DOM), including background color, canvas, or image. FP helps developers understand if anything unexpected is happening during page-rendering.
 
 ### First Contentful Paint (FCP)
 
-[First Contentful Paint (FCP)](https://web.dev/fcp/) measures the time for the first content to render in the viewport. This may be in any form from the document object model (DOM), such as images, SVGs, or text blocks. FCP frequently overlaps with First Paint (FP). FCP helps developers understand how long it takes before the user sees any content change on the page.
+[First Contentful Paint, (FCP)](https://web.dev/fcp/) measures the amount of time it takes for the first content to render in the viewport. This may be in any form from the document object model (DOM), including images, SVGs, or text blocks. FCP frequently overlaps with First Paint (FP). FCP helps developers understand how long it takes before the user sees any content change on the page.
```

Comment:
```suggestion
[First Contentful Paint (FCP)](https://web.dev/fcp/) measures the amount of time it takes for the first content to render in the viewport. This may be in any form from the document object model (DOM), including images, SVGs, or text blocks. FCP frequently overlaps with First Paint (FP). FCP helps developers understand how long it takes before the user sees any content change on the page.
```